src/info/mod.rs

```diff
@@ -417,6 +417,13 @@ mod tests {
             expected_regex,
             simple_info_str
         );
+
+        let v = serde_json::to_value(info).unwrap();
+        let expected_json: serde_json::Value = serde_json::from_str(
+            &include_str!("../../tests/json/test_verilog_repo.serialize.json"),
```

Comment:
Super-small nitpick: since this is basically a snapshot test, so maybe use this name?
```suggestion
            &include_str!("../../tests/json/test_verilog_repo.snapshot.json"),
```

Reply:
Sure thing, will change the name to snapshot.